README.md

```diff
@@ -17,18 +17,18 @@ WooriDB is an (EXPERIMENTAL) immutable time serial database. This project is hug
 
 ## Usage
 * Responses are in `Ron` format, support for `JSON` and `EDN` will be done later by using features.
-* For now only persistent local memory is used. Support for `S3`, `PsotgresDB` and `DynamoDB` will be done later by using features.
+* For now only persistent local memory is used. Support for `S3`, `PostgresDB` and `DynamoDB` will be done later by using features.
 * More info at **TODOS**.
 
 ## Transactions:
-> **Remainder**
+> **Reminder**
```

Comment:
Not sure about this one (remainder is a valid word), but looking at the context this seems more correct.